crates/types/src/serde.rs

```diff
@@ -31,9 +31,11 @@ pub fn parse_state_diffs(data: &[BigUint]) -> Vec<ContractUpdate> {
         let info_word = &data[i];
         i += 1;
 
+        let (class_flag, nonce, number_of_storage_updates) = extract_bits_v2(&info_word);
```

Comment:
are we still using `extract_bits`? if not, we can just change that function instead of creating v2